src/Sylius/Bundle/CoreBundle/Provider/DefaultTaxZoneProvider.php

```diff
@@ -20,23 +20,10 @@
 class DefaultTaxZoneProvider implements ZoneProviderInterface
```

Comment:
I think we could leave it as it is for now, as it used as **default** tax zone provider and it could be easily replaced by some other implementation, not necessarily based on channel.